devdocs/orientation-guide.md

```diff
@@ -0,0 +1,131 @@
+<center>![](images/teammates_heading.png) </center>
+
+
+# **Contributor Orientation Guide**
+
+
+
+We welcome contributions from anyone, in particular, students (see [here](https://www.google.com/url?q=http://teammatesv4.appspot.com/about.html&sa=D&ust=1458427634357000&usg=AFQjCNGIbS5nfk0HxVCMSqPXXmcaOSz47w) for a list of our contributors). One of the main objectives of TEAMMATES is to help students get experience in a OSS production environment. Here are some information that might be useful to would-be contributors.
+
+
+
+[Knowledge required](#h.abftcg92xk1k)
+
+[Workload](#h.8yn20iebwilp)
+
+[Getting started](#h.4es3v3yquxal)
+
+[Contacting us](#h.5lngkuxtl5mo)
+
+* * *
+
+
+
+## Knowledge required
+
+Although TEAMMATES uses many tools and technologies, you need not know all of them before you can contribute. The diagram below shows which technologies you need to learn to contribute in different roles. As you can see, some roles don't need learning any technologies at all. Note that these role aren't fixed or formally assigned. It is simply for guidance only.
+
+<center>
```

Comment:
`<center>...</center>` is not recognized in markdown. You should remove all such instances.